indicator_templates/quarto/3_societal_impact/introduction_societal_impact.qmd

```diff
@@ -0,0 +1,73 @@
+---
+author:
+    - name: T. Venturini
+      orcid: 0000-0003-0004-5308
+      affiliations:
+      - ref: cis-cnrs
+      - ref: ugen
+
+affiliations:
+- id: cis-cnrs
+  name: Centre national de la recherche scientifique
+  department: Centre for Internet and Society
+  city: Paris
+  country: France
+- id: ugen
+  name: University of Geneva
+  city: Geneva
+  country: Switzerland
+---
+
+# Uptake in and impact on to societal issues {#uptake-in-and-impact-on-to-societal-issues .unnumbered}
+
+<div>
+
+## History
+
+| Version | Revision date | Revision    | Author       |
+|---------|---------------|-------------|--------------|
+| 1.0     | 2024-11-14    | First draft | T. Venturini |
+
+</div>
+
+## Open Science Beyond Academia: An Exploration of Societal Uptake and Impact
+
+Open Science is increasingly recognized as essential for advancing academic research and industrial innovation. Understanding the full impact of Open Science, however, requires looking beyond the boundaries of academia and Industry examining its utilization in various societal sectors. This section of the handbook considers how diverse stakeholders engage with Open Science, the challenges in measuring its influence, and the diverse approaches being taken to assess its real-world effects.
+
+Societal impacts are arguably the trickiest impact to assess. Decades of research in the social sciences have shown that direct causal explanations are at best simplistic and at worst misleading. At least three excellent reasons exist to be wary of the very idea of *societal impacts*:
+
+-   Society is, by definition, a sort of residual and all-inclusive category which encompasses all dynamics that cannot be fitted into more specific sectors. As a consequence, societal phenomena are extremely complex and always associated with a multiplicity of causes. Forcedly separating the effect of open science from the other influencing factors simply makes it impossible to understand the issue at hand.
```

Comment:
I'm not sure whether I understand your last sentence. Your observation that societal phenomena are complex and have a multiplicity of causes seems spot on. But why should the effect of open science be forcedly separated from the other factors? It would simply be one of the (potentially) relevant factors in that more complex web of multiple causes, would it not?